docs/modules/plugins/pages/plugin-azure-functions.adoc

```diff
@@ -0,0 +1,67 @@
+= Azure functions Plugin
+
+The plugin provides functionality to interact with https://azure.microsoft.com/en-us/services/functions/.
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-azure-functions', version: '{current-version}')
+----
+
+== Configuration
+
+=== Authentication
+
+By default, plugin's authentication depends on correct configure of following environment variables.
```

Comment:
`plugin's authentication depends on correct configure of following environment variables` ->
`authentication process relies on the configuration of the following environment variables`